elpis/endpoints/model.py

```diff
@@ -28,8 +29,12 @@ def run(cmd: str) -> str:
 def new():
     kaldi: KaldiInterface = app.config['INTERFACE']
     m = kaldi.new_model(request.json["name"])
+    print(app.config['CURRENT_DATABUNDLE'])
```

Comment:
We should perhaps consider using `logging` over printing for this kind of thing at some stage in the future, but also not essential.